smithy-model/src/test/resources/software/amazon/smithy/model/errorfiles/validators/deprecated-shape/deprecated-shape.errors

```diff
@@ -1,4 +1,4 @@
-[WARNING] smithy.example#Test$foo1: Member targets a deprecated shape, smithy.example#Foo1 | DeprecatedShape
-[WARNING] smithy.example#Test$foo2: Member targets a deprecated shape, smithy.example#Foo2 (since 2.0) | DeprecatedShape
-[WARNING] smithy.example#Test$foo3: Member targets a deprecated shape, smithy.example#Foo3. hello (since 2.0) | DeprecatedShape
-[WARNING] smithy.example#Test: Applies a deprecated mixin, smithy.example#DeprecatedMixin | DeprecatedShape
+[WARNING] smithy.example#Test$foo1: Member targets a deprecated shape, smithy.example#Foo1 | DeprecatedShape.MEMBER_TARGET.smithy.example#Foo1
```

Comment:
Does the kind of member relationship matter here? Seems like it could just refer to the deprecated target instead. Exposing MEMBER_TARGET et al seems like it's exposing too much of an implementation detail in the ID.

Reply:
That was just to make it easier to suppress all violations regarding deprecated members or all violations regarding deprecated mixins separately. But yeah there's maybe not too much value there so I'll remove those categories on the next commit